Arduino/SensorLogic/SensorLogic.ino

```diff
@@ -118,8 +119,35 @@ void RFIDloop()
     Serial.print(RC522.serNum[i],DEC);
     //Serial.print(RC522.serNum[i],HEX); //to print card detail in Hexa Decimal format
     }
+    
     Serial.println();
     Serial.println();
+    Servoloop();
   }
   delay(1000);
 }
+
+void Servosetup()
+{
+  myservo.attach(7);
+  myservo.write(pos);
+}
+
+void Servoloop()
+{
+  myservo.attach(7);
+  for(pos = 0; pos <= 180; pos += 1)
+  {
+    myservo.write(pos);
+    Serial.println(myservo.read());
+    delay(100);
+  }
+  for(pos = 180; pos >= 0; pos -= 1)
+  {
+    myservo.write(pos);
+    Serial.println(myservo.read());
+    delay(100);
+  }
+  myservo.write(90);
+  myservo.detach();
+}
```

Comment:
This starts by attaching the Servo to pin 7 which turns on the Servo motor. It then alternates from one direction to the other and flips back around. This is seen as it slows down and revs up in speed.

It finally sets the servo to the no movement position and detaches the servo to turn it off. Setting the position to 90 is unnecessary since detaching the servo will completely shut the servo down. However, detaching is necessary since the setting the servo at 90 doesn't actually turn it off. It leaves it at a struggling position where the servo is still buzzing, but it can't move.